partitions.csv

```diff
@@ -0,0 +1,5 @@
+# ESP-IDF Partition Table
```

Comment:
May want to add the partitions.csv setup as default in the sdkconfig.default setup, otherwise I'd probably remove this

Reply:
added default partitions with 7d0ca95